### PR TITLE
Roles manager page will now reset after filtering the competitors.

### PR DESCRIPTION
**Pull request by druidalek**

### CODE REVIEW

File: src/components/Competition/RolesManager/RolesManager.js [hunk 1]
```diff
@@ -88,7 +93,7 @@ const RolesManager = ({ wcif, onWcifUpdate }) => {
                 <TextField
                   label="Search"
                   value={searchString}
-                  onChange={event => setSearchString(event.target.value)}
+                  onChange={event => handleSearchStringChange(event.target.value)}
```
Comment (jonatanklosko) [inline src/components/Competition/RolesManager/RolesManager.js]: Nit: for the sake of convention, I'd suggest passing the whole event to the handler and just do `onChange={handleSearchStringChange}` =)

**Review by jonatanklosko — COMMENTED** [review]
Thanks for contributing :tada: Just one comment ^^

File: src/components/Competition/RolesManager/RolesManager.js (hunk 1 above)
Comment (druidalek) [reply src/components/Competition/RolesManager/RolesManager.js]: Thanks for the advice. React is really new to me, so I'm not familiar with best practices yet.

**Review by jonatanklosko — APPROVED** [review]
Thanks!

**Pull request merged**